lib/modules/platform/azure/index.ts

```diff
@@ -288,7 +288,6 @@ export async function getPr(pullRequestId: number): Promise<Pr | null> {
     .filter((label) => label.active)
     .map((label) => label.name)
     .filter(is.string);
-  azurePr.hasReviewers = is.nonEmptyArray(azurePr.reviewers);
```

Comment:
If the only change for Azure is removing this logic, isn't this removing functionality for no reason?

Reply:
Few providers already haev `reviewers` property, and few of them do not. You can see here on that line that reviewers are already present in the PR structure.

Reply:
OK, I. missed that `reviewers` would be passed up anyway, so I guess it's redundant